create_jenkins_job.py

```diff
@@ -67,6 +67,7 @@ def main(argv=None):
     args = parser.parse_args(argv)
 
     data = {
+        'build_discard': dict(),
```

Comment:
Nitpick: use `{}` instead.

Reply:
Would have expected `{}` to create an empty set rather than an empty dict but TIL.